composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Allows you to get values for read only settings that are stored in settings.php",
     "type": "drupal-drush",
     "require": {
-        "drush/drush": "^8.1"
+        "drush/drush": "^8.1 || ^9@beta"
```

Comment:
Let's only support for ^9@beta here. If someone needs support for Drush:8,  ^0.2 is the way to go. That also means - remove old command definition.